docs/src/templates/pages/documentation/general/who-we-support.hbs

```diff
@@ -6,7 +6,7 @@ docs: true
 
 navgroup: documentation
 navsub: docs-general
-navactive: who-we-support
+navactive: whom-we-support
```

Comment:
NIT: Filename should change as well.

Reply:
Damn, I thought I'd got everything. Good spot 👀